### PR TITLE
[kie-issues#51] Prepare modules structure for quick benchmarks

### PR DESCRIPTION
**Pull request by baldimir**
Solves https://github.com/kiegroup/kie-issues/issues/51

This is a basic structure that creates a separate benchmarks module for quick benchmarks, that could be later used in PR checks etc. Currently the module is empty, I want to work on the benchmarks continuously in following PRs in the near future. Or should I propose this now without the drools-benchmarks-quick module and add it later? 

**Reviewers please be aware that this has automation implications.** If you need to see the structure, I think it can be better visible if you go directly to my [branch from which this PR is created](https://github.com/baldimir/kie-benchmarks/tree/drools-smoke-benchmarks).

### CODE REVIEW

File: drools-benchmarks-parent/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Drools benchmarks
+==========================
+
+The benchmarks are written using [JMH](http://openjdk.java.net/projects/code-tools/jmh/).
+
+This module contains multiple Maven modules. The instructions how to run individual benchmark modules are described in the README files of the modules:
```
Comment (danielezonca) [inline drools-benchmarks-parent/README.md:6]: Can you add some info also for `drools-benchmarks-common`?

File: drools-benchmarks-parent/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+- drools-benchmarks
+  - Variety of benchmarks, benchmarks are grouped by package names. May have longer run time.  
+- drools-benchmarks-quick
+  - Quick benchmarks that are supposed to be used for initial sanity measurements.
```
Comment (danielezonca) [inline drools-benchmarks-parent/README.md:10]: We will probably need to be a bit more precise with `quick` definition :)
We can also see if `OptaPlanner` team is interested in a similar approach

**Review by danielezonca — APPROVED** [review]

File: drools-benchmarks-parent/README.md (hunk 1 above)
Comment (baldimir) [reply drools-benchmarks-parent/README.md:6]: Fixed.

File: drools-benchmarks-parent/README.md (hunk 2 above)
Comment (baldimir) [reply drools-benchmarks-parent/README.md:10]: We can align the description as the quick benchmarks will be added in future PRs. Would that be ok?

Comment (danielezonca) [reply drools-benchmarks-parent/README.md:10]: Fine for me 👍

**Review by hellowdan — APPROVED** [review]

**Pull request merged**